src/app/components/list Projects/Listsprojects.tsx

```diff
@@ -0,0 +1,74 @@
+"use client";
+import React, { useState } from "react";
+import {
+  List,
+  ListItem,
+  ListItemText,
+} from "@mui/material";
+import { AiOutlineDelete } from "react-icons/ai";
+import styles from "./style.module.scss";
+import ConfirmDelete from "../confirmDelete/ConfirmDelete";
+import { useRouter } from "next/navigation";
+import { deleteProject, getAllProjects } from "@/app/Service/ProjectServices";
+import { useMutation, useQuery, useQueryClient } from "react-query";
+
+function Listsprojects({ AllprojectsProps }: { AllprojectsProps: any }) {
+  const [openDeleteDialog, setopenDeleteDialog] = useState(false);
+  const router = useRouter();
+  const queryClient = useQueryClient();
+  const { data: allprojects } = useQuery("Allprojects", getAllProjects);
+
+  //Delete Project
+  const mutation = useMutation({
+    mutationFn: deleteProject,
```

Comment:
don't do that always destructure the mutation and always if the function take param make it like that 
`mutationFn: (projectId) => deleteProject(projectId)`